openscenario/openscenario_utility/openscenario_utility/conversion.py

```diff
@@ -128,7 +128,10 @@ def from_yaml(keyword, node):
                 #
                 # => @tag: { ... }
                 #
-                result["@" + tag] = str(value)
+                if isinstance(value, bool):
+                    result["@" + tag] = str(value).lower()
+                else:
+                    result["@" + tag] = str(value)
```

Comment:
I debated both solutions here in #1646, and ultimately chose to add one extra processing and conversion. Adding a boolean if-statement here, people may forget about the processing here long into the future. Adding more comments adds additional functionality to this function, so it might be better to separate them.

BTW, this modification doesn't seem to have much to do with this PR topic, and then my contribution will be lost :(

Reply:
I'm more than happy to rebase this on another separate solution. The reason the fix is included here is that it was a blocker for me getting working builds with the new Dockerfiles and I had not yet seen that another PR was open with a proposed fix.

Reply:
Thanks, I'sd appreciate it if you are willing to rebase the solution. For reference, here is the PR addressing the fix: #1648 (we opened them very closely :). It’s likely to be merged, so please feel free to leave this partto that PR if you want.